app/models/review.rb

```diff
@@ -0,0 +1,7 @@
+class Review < ActiveRecord::Base
+  belongs_to :restaurant
+  belongs_to :user
+  validates :body, presence: true
+  validates :rating, presence: true
```

Comment:
you should also validate that the rating is one of the defined elements in the `RATINGS` constant. Place the `RATINGS` constant above the validations, and use the `:inclusion` option